public/data/pranay202.json

```
@@ -0,0 +1,67 @@
+{
+  "name": "Pranay Kharabe",
+  "type": "personal",
+  "bio": "I'm a full stack MERN developer and an open-source enthusiast specializing in building exceptional digital experiences. Currently focused on building responsive full stack web applications.",
+  "avatar": "https://github.com/pranay202.png",
+  "links": [
+  {
+      "name": "Connect on Linkedin",
```

I recommend capitalizing the `i` here. What do you think?
```suggestion
      "name": "Connect on LinkedIn",
```